modules/serverless-installing-cli-linux-ibm-power_tarball.adoc

```diff
@@ -0,0 +1,39 @@
+[id="installing-cli-linux_ibm_power_tarball_{context}"]
+
+== Tarball Installation
+
+For Linux distributions, you can download the CLI directly as a `tar.gz` archive.
+
+
```

Comment:
extra line, remove.
Same comments for above section apply, please remove all unnecessary line spacing